support/build/config-submenu.sh

```diff
@@ -98,6 +110,39 @@ else
 	exec 7>&1
 fi
 
+#
+# GENERATE LIST OF FINAL LIBRARY PATHS
+#
+LIBS=()
+for ARG_PATH in "${ARG_PATHS[@]}" "$@"; do
+	# Remove trailing slashes
+	ARG_PATH="${ARG_PATH%/}"
+
+	# Does Config.uk exist?
+	CONFIG_UK="${ARG_PATH}/Config.uk"
+	if [ ! -f "${CONFIG_UK}" ]; then
+		if [ $OPT_QUIET -ne 0 ]; then
+			echo "Could not find \"Config.uk\" under \"${ARG_PATH}\". Skipping..." 1>&2
+		fi
+		continue
+	fi
+
+	DONT_INCLUDE=1
+	for ARG_EXCLUDE in "${ARG_EXCLUDES[@]}"; do
+		if [ "${ARG_PATH}" = "${ARG_EXCLUDE}" ]; then
+			if [ $OPT_QUIET -ne 0 ]; then
```

Comment:
Theoretically all variables should be escaped with `${}` to prevent globbing.

Also I remember reading that upper case variables are usually reserved for env vars and 'global' variables, but it is just a convention.

Reply:
`shellscript` is actually not bad. We can discuss next maintainer meeting if we want to use it. I'd suggest to update our scripts based on that output.